Makefile

```diff
@@ -0,0 +1,38 @@
+.PHONY: format lint coverage isort-format black-format
```

Comment:
Some of the rules are not listed there (and I don't see `coverage`). Is it what is expected ?

Also, instead of having a single .PHONY with all the rules, I think it is easier to prefix each rule with its own .PHONY to keep track of which are used and which are not 🙂 .

```Makefile
.PHONY: rule1
rule1:
    echo rule1

.PHONY: rule2
rule2:
    echo rule2
```

Reply:
Having one PHONY per rule seems a bit overkill to me as it will kind of make the file becomes very hard to read...